### DISCUSSION TITLE
a session with random exercises from a bunch of different lessons.

### DISCUSSION THREAD

**Discussion by ghost** [SethClydesdale/genki-study-resources#148] [Ideas]
I always thought genki's weakest point were the exercises because of one thing and one thing only. when you learn how to conjugate i-adjectives in the past affirmative, simple right? just takes out the い and replace it with かった. that virtually never changes. you only need to do an example or two to get the pattern. what's hard is remembering that pattern. And as most of you probably know, dedicating 5 minutes to remember a kanji in one good would be far less effective than splitting that 5 minutes over the period of a week or so.

Of course, a full-fledged SRS would be invaluable but merely giving exercises randomly (I think) would be tremendously helpful. we just select the lessons we want to practice, whether we want to practice grammar or vocab or both or something else, and it gives exercises from ALL of them.

**Comment by SethClydesdale** [comment]
Hello!

I think it is a nice idea, though currently I'm occupied with other things to focus on new implementations for this project. It may not be exactly as you hoped, but there IS a random exercise button in the exercise list on the [top left](https://user-images.githubusercontent.com/10122549/132564253-4fc3957a-f3ac-4074-a4ee-bf8ebdc13f8f.PNG). You can localize the pool of random exercises to the current lesson via the site settings, too (cog at the bottom of the page or CTRL+ALT+M > random exercise range).
